crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -42,13 +45,21 @@ use super::{
 };
 
 #[derive(Clone)]
-pub(crate) struct TransactionBlock {
-    /// Representation of transaction data in the Indexer's Store. The indexer stores the
-    /// transaction data and its effects together, in one table.
-    pub stored: StoredTransaction,
-
-    /// Deserialized representation of `stored.raw_transaction`.
-    pub native: NativeSenderSignedData,
+pub(crate) enum TransactionBlock {
```

Comment:
nit: Some doc comments here would be helpful, summarising where these variants come from and what they do and don't come with.